src/vs/workbench/api/browser/mainThreadEditors.ts

```diff
@@ -136,14 +138,16 @@ export class MainThreadTextEditors implements MainThreadTextEditorsShape {
 	async $tryShowTextDocument(resource: UriComponents, options: ITextDocumentShowOptions): Promise<string | undefined> {
 		const uri = URI.revive(resource);
 
+		let notebookFileTypes = this._notebookService.getSupportedFileExtensions().map(s => s.toLowerCase());
```

Comment:
What is the general rule around editing vs files? Does each line need a {{SQL CARBON EDIT}}?